### PR TITLE
ref(seer grouping): Check rate limits before seer ingestion grouping call

### PR DESCRIPTION
**Pull request by lobsterkatie**
This adds both a similarity-service-wide rate limit check and a per-project rate limit check before calling the Seer similarity service during ingestion. The rate limits are based on two new options, `seer.similarity.global-rate-limit` and `seer.similarity.per-project-rate-limit`. 

Note to reviewers: Lacking good intuition about what would make for a reasonable limit, I copied the default values from the Seer severity rate limits. Happy to adjust them, though.

### CODE REVIEW

**Review by jangjodi — COMMENTED** [review]
lgtm, and the rate limits make sense to me. let's wait for Josh to review before merging though

**Review by JoshFerge — APPROVED** [review]
makes sense, and we can bump the rate limit numbers up once we roll out if deemed safe.

**Pull request merged**
